test/core/transport/chttp2/too_many_pings_test.cc

```diff
@@ -675,8 +676,6 @@ void PerformCallWithResponsePayload(grpc_channel* channel, grpc_server* server,
   op->op = GRPC_OP_SEND_STATUS_FROM_SERVER;
   op->data.send_status_from_server.trailing_metadata_count = 0;
   op->data.send_status_from_server.status = GRPC_STATUS_OK;
-  grpc_slice status_details = grpc_slice_from_static_string("xyz");
```

Comment:
I don't understand why this is changing

Reply:
One of the behavior changes with this experiment is that we'll no longer propagate the message if the status is OK, since we'll now be using the normal behavior of `absl::Status`.  But we had a lot of tests that were passing OK status with a message and expecting the message to be passed through.  A lot of the test changes in this PR are to either stop checking the message or to use a status code other than OK, so that the tests will pass regardless of whether the experiment is enabled.